src/Microsoft.AspNetCore/WebHost.cs

```diff
@@ -182,6 +182,16 @@ public static IWebHostBuilder CreateDefaultBuilder(string[] args)
                     logging.AddConsole();
                     logging.AddDebug();
                 })
+                .ConfigureServices((hostingContext, services) =>
+                {
+                    var hosts = hostingContext.Configuration["AllowedHosts"]?.Split(new[] { ';' }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Hmm hardcoded constants look suspicious. Why isn't `AllowedHosts` defined as a static string somewhere?

Reply:
It's only used in this one spot. There are several of those in this method.